src/current/cockroachcloud/production-checklist.md

```diff
@@ -66,6 +66,20 @@ When several transactions try to modify the same underlying data concurrently, t
 
 To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
+## Configure node shutdown settings
```

Comment:
this section LGTM from a docs editing POV but I defer to @carloruiz for technical review

Reply:
I'd move this section below `Keeping connection current` since the desired value for `connection_wait` should be set equal to or above the max lifetimes set on their connections.

Reply:
On further read, I think this section should be folded under "Keeping connections current"

Reply:
Done, created a new Networking considerations heading and moved Keeping Connections Current (merged with the shutdown settings section) and Network Authorization into it.